lib/node_modules/@stdlib/ndarray/base/some-by/lib/0d.js

```diff
@@ -0,0 +1,84 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MAIN //
+
+/**
+* Tests whether at least `n` elements in an ndarray pass a test implemented by a predicate function.
+*
+* @private
+* @param {Object} x - object containing ndarray meta data
+* @param {ndarrayLike} x.ref - reference to the original ndarray-like object
+* @param {string} x.dtype - data type
+* @param {Collection} x.data - data buffer
+* @param {NonNegativeIntegerArray} x.shape - dimensions
+* @param {IntegerArray} x.strides - stride lengths
+* @param {NonNegativeInteger} x.offset - index offset
+* @param {string} x.order - specifies whether `x` is row-major (C-style) or column-major (Fortran-style)
+* @param {PositiveInteger} n - number of elements
```

Comment:
s/PositiveInteger/NonNegativeInteger/ here and elsewhere. While `0` and `1` are equivalent, no reason, IMO, to exclude `0` as an "allowed" value.

Reply:
Actually, thinking about this, if `n = 0`, we'd just add a line in `main.js` to automatically return `true`, as it is trivially true that at least `0` elements satisfy the predicate. In which case, we can leave `PositiveInteger` here and elsewhere.